### ISSUE TITLE
Design

### ISSUE THREAD

**Issue by susanjrobertson** [refreshpdx/website#23]
We've got wireframes, so maybe do some _rough_ designs that we can take into implementation?

**Comment by dalla777** [comment]
Here's where I'm at for the homepage right now... the other thing I had going was pretty meh.

I thought we should probably include something about sponsorship in the footer, and think about where a sponsor logo may go if we need to add it to the site.

![refresh_design_2](https://cloud.githubusercontent.com/assets/1205728/21036366/338d71b2-bd7a-11e6-9a14-b109597c87a7.png)

**Comment by fajalar** [comment]
I like where this is going. Wonder about the light green background on the primary event and what it would look like instead as bg for the upcoming events and have the primary be a white bg. 

The logo looks like it will make a cool die-cut sticker. :) Assume it would drop down to just the leaf on small viewports?

**Comment by dalla777** [comment]
Yeah will have to work on all the sizes, just wanted to check in with that idea first. Good call on the color swap. Also I might introduce a background pattern in the light blue area

**Comment by fajalar** [comment]
I showed this to Claire. She sent back feedback via email (I fwd'd it to her via email so she could do it in her own time.) As with all feedback there's something to it and it's way off. :)

~~~
1 The white space around the RPDX looks cartoony. Maybe a circle or a leaf shape? Or a different sharper font?

2 The colors are a bit gloomy or dull. I think it might be the neutral green behind the speaker, or how dark the top green is. I'm not advocating for lime or kelly, but something a little more lively might be... livelier.

3 The white space between the two blues feels in need of a containing border.

Generally it's not bad, but doesn't grab my attention.

**Comment by dalla777** [comment]
I would agree with the cartoony part, I was undecided about the logo, I will work on how that looks. I also agree with the colors being gloomy, but I went with the palette we chose. If you guys don't object I can brighten the green up a bit. "doesn't grab my attention." I'm not sure that we need it to be flashy, so I'm mostly going to ignore this comment.

**Comment by fajalar** [comment]
word.
